cmds/tcz/tcz.go

```diff
@@ -201,10 +207,18 @@ func installPackage(tczName string, deps map[string]bool) error {
 func setupPackages(tczName string, deps map[string]bool) error {
 	debug("setupPackages: @ %v deps %v\n", tczName, deps)
 	for v := range deps {
-		cmdName := strings.Split(v, ".")[0]
+		cmdName := strings.Split(v, filepath.Ext(v))[0]
 		packagePath := filepath.Join("/tmp/tcloop", cmdName)
+
+		if _, err := os.Stat(packagePath); err == nil {
+			debug("PackagePath %s exists, skipping mount", packagePath)
+			continue
+		}
+
 		if err := os.MkdirAll(packagePath, 0700); err != nil {
-			l.Fatal(err)
+			l.Printf("overwriting %s", packagePath)
```

Comment:
I'm still not sure we need this print. What does it mean? We're not overwriting anything at this point, since we know nothing was there?